wire-ios-request-strategy/Sources/Synchronization/Decoding/EventDecoder+MLS.swift

```diff
@@ -84,12 +84,16 @@ extension EventDecoder {
                     }
 
                     if let mlsService, updateEvent.source == .webSocket {
-                        do {
-                            try await mlsService.commitPendingProposals()
-                        } catch {
-                            WireLogger.mls.error("failed to commit pending proposals: \(String(describing: error))")
+                        // It's not uncommon to receive multiple proposals in a row for a group, and attempting to
+                        // commit only one of them will result in mls-commit-missing-references and uncessary re-tries
+                        // so we defer committing them for later.
+                        WaitingGroupTask(context: context) {
+                            do {
+                                try await mlsService.commitPendingProposals()
+                            } catch {
+                                WireLogger.mls.error("failed to commit pending proposals: \(String(describing: error))")
+                            }
```

Comment:
How does this defer committing for later?

Reply:
Maybe defer is the wrong word here, but we don't want the event processing/decryption queue to wait for this compete.